### ISSUE TITLE
Reading multichannel Zeiss LSM files

### ISSUE THREAD

**Issue by mlaasmaa** [cgohlke/tifffile#269]
Dear Christoph,

First of all, thank you for the tifffile package it's very useful.

I'm working with Zeiss LSM line-scan (time and space) images with 2 channels. The problem is that when I use TiffFile to read the files it somehow stacks them on one axis (T); see figure below. However, when I convert LSM to TIFF using ImageJ TiffFile reads it in correctly. So, I was wondering whether I was doing something wrong. The tifffile package version that I'm using is 2024.8.24 via ```pip```. The LSM and respective TIF file can be downloaded here: https://www.dropbox.com/scl/fi/d5np2s41d8covqxuub08p/example.zip?rlkey=27j7ee5cnzcnxv68vkajhd9t4&st=i4627sk2&dl=0

The example code:
```python
import numpy as np
import pylab as plt
import tifffile

print(tifffile.__version__)

fig = plt.figure()

for i, suffix in enumerate(["lsm", "tif"]):
    with tifffile.TiffFile(f"example.{suffix}") as tif:
        if tif.is_lsm is True:
            image = np.squeeze(tif.asarray())
        else:
            image = np.squeeze(tif.asarray())
            image = np.swapaxes(image, 0, 1)

        for channel in range(2):
            ax = fig.add_subplot(1, 4, 2 * i + channel + 1)
            ax.imshow(image[:, channel, :], aspect="auto", vmax=50)
            ax.set_title(f"{suffix} - channel:{channel}")

plt.show()
```
and how it looks
![image](https://github.com/user-attachments/assets/6bc3cf42-4f5d-4230-8423-8286ad861cf1)

I appreciate it if you could help me with that.

Kind regards,
Martin

**Comment by cgohlke** [comment]
Thank you for sharing the file. The LSM metadata says it's a "TCYX" scan, but the data is stored as "CTX". This needs to be special-cased. The point scan LSM files for testing so far had only single channels where this did not matter.

As a workaround this specific issue, use `tif.asarray(key=0)` to get the correct values in "CTX" order.

**Comment by cgohlke** [comment]
This should be fixed in tifffile-2024.8.28 (about to be released), which returns CTX axes and correct values for your file.

The dimensions returned for LSM series are now based on the LSM ScanType and newer LSM documentation. This is a breaking change.

**Comment by mlaasmaa** [comment]
Thank you so much for the quick reply and the fix.
I've now installed tifffile-2024.8.28, and reading multichannel LSMs works beautifully.
